tox.ini

```diff
@@ -92,7 +91,6 @@ description = Run all style and file checks with pre-commit
 deps =
     pre-commit
 commands =
-    pre-commit install-hooks
```

Comment:
I added this separately to make it more obvious if an issue occurred during install vs run, also just to be explicit about things. Don't mind if you still want to delete it tho.

Reply:
Fair enough, I will add it back.